plugins/modules/source_control/github/github_tag.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: Ansible Team
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: github_tag
+short_description: Create annotated GitHub tags for use with releases
+description:
+    - Allows for creation of annotated tags, by default a GitHub release only create lightweight tags.
+    - Use this module with the `github_release` module to created a release linked to a annotated tag.
+options:
+    url:
+        description:
+            - Custom GitHub URL for self-hosted enterprise versions.
+        type: str
+    token:
+        description:
+            - GitHub access Token for authenticating.
+            - Mutually exclusive with C(password).
```

Comment:
```suggestion
            - Mutually exclusive with I(password).
```
Use I(...) for option names, and C(...) for option values.